### PR TITLE
runtime: add some debug logs

### PR DESCRIPTION
**Pull request by samuelkarp**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: runtime/service.go [hunk 1]
```diff
@@ -201,6 +217,7 @@ func (s *service) Create(ctx context.Context, request *taskAPI.CreateTaskRequest
 	}
 	// TODO: should there be a lock here
 	if !s.agentStarted {
+		log.G(ctx).WithField("id", request.ID).Debug("calling startVM")
 		client, err := s.startVM(ctx, request, firecrackerConfig)
 		if err != nil {
 			log.G(ctx).WithError(err).Error("failed to start VM")
```
Comment (sipsma) [inline runtime/service.go]: nit: can also include the request.ID here too now

File: runtime/service.go [hunk 2]
```diff
@@ -251,17 +268,18 @@ func (s *service) proxyStdio(ctx context.Context, stdin, stdout, stderr string,
...
 	f, err := fifo.OpenFifo(ctx, path, syscall.O_RDWR|syscall.O_NONBLOCK, 0700)
 	if err != nil {
-		log.G(ctx).WithError(err).Error("error opening fifo")
+		log.G(ctx).WithError(err).WithField("path", path).Error("error opening fifo")
 		return
 	}
 	conn, err := vsock.Dial(cid, port)
 	if err != nil {
-		log.G(ctx).WithError(err).Error("unable to dial agent vsock")
+		log.G(ctx).WithError(err).WithField("path", path).Error("unable to dial agent vsock")
```
Comment (sipsma) [inline runtime/service.go]: nit: could include something like `unable to dial agent vsock IO port` to distinguish from the other vsock dial we do

File: runtime/service.go (hunk 1 above)
Comment (samuelkarp) [reply runtime/service.go]: Done

File: runtime/service.go (hunk 2 above)
Comment (samuelkarp) [reply runtime/service.go]: Done

**Pull request merged**